contrib_bots/bots/commute/commute.py

```diff
@@ -84,14 +84,15 @@ def triage_message(self, message, client):
         # ups for own follow ups!
         if message['display_recipient'] == 'commute':
             return False
-        is_commute = original_content.startswith('@commute')
+        is_commute = (original_content.startswith('@commute') or
+                      message['is_mentioned'])
         return is_commute
 
     # adds API Authentication Key to url request
     def get_api_key(self):
-        # commute_bot.config must have been moved from
-        # ~/zulip/contrib_bots/bots/commute_bot/commute_bot.config into
-        # /commute_bot.config for program to work
+        # commute.config must have been moved from
```

Comment:
`must have been moved` is a tad complicated, maybe `must be moved` would be better here?